src/main/java/pl/agh/edu/master_diet/core/model/shared/SearchResult.java

```diff
@@ -0,0 +1,18 @@
+package pl.agh.edu.master_diet.core.model.shared;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Setter
```

Comment:
You're missing ```@Builder```  - it should be in here cause of ```@AllArgsConstructor``` and ```@NoArgsConstructor```

Reply:
Done.